tests/routes/test_sessions.py

```
@@ -563,14 +575,14 @@ def test_set_session_peers(client: TestClient, sample_data: tuple[Workspace, Pee
     assert data["items"][0]["id"] == peer2_name
 
 
-def test_set_session_peers_with_limit(
+def test_set_session_peers_with_observer_limit(
```

Ruff (0.12.2)

Missing return type annotation for public function `test_set_session_peers_with_observer_limit`

Add return type annotation: `None`

(ANN201)